.npmrc

```diff
@@ -1,8 +1,10 @@
+enable-pre-post-scripts=true
 hoist-pattern[]=*eslint*
 hoist-pattern[]=*babel*
 hoist-pattern[]=@emotion/*
 hoist-pattern[]=postcss
 hoist-pattern[]=pug
 hoist-pattern[]=source-map-support
 hoist-pattern[]=ts-node
+shamefully-hoist=false
```

Comment:
I think this line can be removed since it's false by default

Reply:
I have `~/.npmrc` set `shamefully-hoist=true`, this ensure this setting is always false for contributors.

Reply:
Is there a reason you changed the default globally? I think it should be a per-project setting, otherwise if set globally, you'd be responsible to _unset_ the config per-project.

Reply:
Because `shamefully-hoist=false` works bad for a lot of my projects.

Reply:
No matter the reason, a specific setting here will be consistent for all contributors.

Reply:
I still think we should remove this. If your projects need `shamefully-hoist=true`, you should set the config per project, otherwise you'd be expecting contributors of your project to have the same global setting.

Reply:
> otherwise you'd be expecting contributors of your project to have the same global setting

That's why we should have this config here.

I always commit a project level `.npmrc` with `shamefully-hoist=true` at the same time.

The global `shamefully-hoist=false` is for my global packages.

Reply:
Fine, just remove this line to not block this PR.